kafka/src/main/java/com/metamx/tranquility/kafka/writer/TranquilityEventWriter.java

```diff
@@ -0,0 +1,231 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.metamx.tranquility.kafka.writer;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.Lists;
+import com.metamx.common.logger.Logger;
+import com.metamx.tranquility.beam.Beam;
+import com.metamx.tranquility.beam.BeamPacketizer;
+import com.metamx.tranquility.beam.BeamPacketizerListener;
+import com.metamx.tranquility.beam.ClusteredBeamTuning;
+import com.metamx.tranquility.druid.DruidBeams;
+import com.metamx.tranquility.druid.DruidDimensions;
+import com.metamx.tranquility.druid.DruidLocation;
+import com.metamx.tranquility.druid.DruidRollup;
+import com.metamx.tranquility.druid.DruidSpatialDimension;
+import com.metamx.tranquility.kafka.model.IngestionSpec;
+import com.metamx.tranquility.kafka.model.TranquilityConfig;
+import com.metamx.tranquility.kafka.util.IngestionSpecHelper;
+import com.metamx.tranquility.typeclass.Timestamper;
+import io.druid.data.input.impl.SpatialDimensionSchema;
+import io.druid.segment.indexing.DataSchema;
+import org.apache.curator.RetryPolicy;
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.CuratorFrameworkFactory;
+import org.apache.curator.retry.ExponentialBackoffRetry;
+import org.joda.time.DateTime;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * An EventWriter that pushes events to Druid through Tranquility.
+ */
+public class TranquilityEventWriter implements EventWriter
+{
+  private static final Logger log = new Logger(TranquilityEventWriter.class);
+  private static final ObjectMapper mapper = new ObjectMapper();
+  private static final RetryPolicy retryPolicy = new ExponentialBackoffRetry(1000, 500, 30000);
```

Comment:
generally, other than `log` we capitalize all the statics. So `MAPPER` and `RETRY_POLICY`